index.js

```diff
@@ -70,7 +70,7 @@ function requestbody(opts) {
     // only parse the body on specifically chosen methods
     if (opts.parsedMethods.includes(ctx.method.toUpperCase())) {
       try {
-        if (opts.json && ctx.is('json')) {
+        if (opts.json && ctx.is('json', 'application/json-patch+json')) {
```

Comment:
I'm thinking we should just borrow the list if JSON content types from koa-bodyparser https://github.com/koajs/bodyparser/blob/master/index.js#L45-L50

Reply:
Does it make sense to use `+json` instead of `['application/json-patch+json', 'application/vnd.api+json']` so it would support any media types that represent a JSON document (see [Structured Syntax Name Suffixes](https://tools.ietf.org/html/rfc6838#section-4.2.8))?

```
if (opts.json && ctx.is('json', '+json', 'application/csp-report'))
```

Or should this lib only handle a few explicit types?

Reply:
@MarkHerhold I just pushed a commit that adds support for the same types as in bodyparser (PR description updated).